ios/FluentUI/People Picker/PeoplePicker.swift

```diff
@@ -146,6 +142,9 @@ open class PeoplePicker: BadgeField {
 
     private let separator = Separator()
 
+    public typealias TokenSetKeyType = PeoplePickerTokenSet.Tokens
+    public var peoplePickerTokenSet: PeoplePickerTokenSet = .init()
```

Comment:
Since `PeoplePicker` inherits from `BadgeField`, `tokenSet` already exists. Open to suggestions on how we want to approach two token sets (one for BadgeField and one for PersonaListView)

Reply:
For `PopupMenu`, I made it assert on trying to access the drawer tokens. It's probably better to keep all the values coming from one tokenSet if possible

```swift
public typealias TokenSetKeyType = PopupMenuTokenSet.Tokens
public typealias TokenSetType = PopupMenuTokenSet
public override var tokenSet: DrawerTokenSet {
  get {
    return popupTokenSet
  }
  set {
    assertionFailure("PopupMenuController tokens must be set through popupTokenSet")
  }
}
public var popupTokenSet: PopupMenuTokenSet = .init()
```

Reply:
Though, I guess that made more sense when overrides were done by setting a custom token class. This should still work, though, since `popupMenu.tokenSet` will return the `popupTokenSet`.